client/landing/stepper/declarative-flow/flows/onboarding/onboarding.ts

```diff
@@ -203,6 +210,15 @@ const onboarding: Flow = {
 				];
 			}
 
+			if ( isMvpOnboarding ) {
+				return [
+					`/overview/${ providedDependencies.siteSlug }`,
+					addQueryArgs( withLocale( `/setup/${ flowName }/plans`, locale ), {
+						siteSlug: providedDependencies.siteSlug,
+					} ),
```

Comment:
I was originally thinking of treating users as free users if they return from the checkout, but after testing it, it does make sense to just return them to the /plans page.

Reply:
Yeah a number of our flows just count it as a failed checkout and proceed without a plan. But now that StepContainerV2 labels the button a "Back" button, it feels even more natural to send them back to plans I think.